### PR TITLE
Fix horizontal sidebar rendering overlapping the active service

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Fix horizontal sidebar rendering overlapping the active service.

#### Motivation and Context
Fixes the horizontal sidebar rendering on top of app. This solves the issue https://github.com/ferdium/ferdium-app/issues/69 and was due (blames) to the PR https://github.com/ferdium/ferdium-app/pull/58 merged to the codebase.

Confirmed to work with:
- [x] Mac
- [x] Linux
- [x] Windows

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes
Fix horizontal sidebar rendering overlapping the active service

### CODE REVIEW

File: src/features/appearance/index.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { reaction } from 'mobx';
 import { DEFAULT_APP_SETTINGS, iconSizeBias } from '../../config';
 
 const STYLE_ELEMENT_ID = 'custom-appearance-style';
+const isWindows = process.platform === 'win32';
```
Comment (vraravam) [inline src/features/appearance/index.ts]: There's a constant already exported for `isWindows` from `environment.ts` - please use that

**Review by vraravam — CHANGES_REQUESTED** [review]
Please also delete trailing spaces in the files.

File: src/features/appearance/index.ts (hunk 1 above)
Comment (vraravam) [reply src/features/appearance/index.ts]: Fixed in latest commit

**Comment by vraravam** [comment]
looks good on macos!
Screenshot 2022-04-30 at 11 00 17 AM

Running this commit:
Screenshot 2022-04-30 at 10 56 43 AM

**Review by vraravam — APPROVED** [review]

**Comment by SpecialAro** [comment]
Tested on Linux as well, all working fine
image

**Pull request merged**